projects/sandbox/datagen/datagen/scripts/timeslide_waveforms.py

```diff
@@ -47,7 +47,60 @@ def main(
     verbose: bool = False,
 ):
     """
-    Generates the waveforms for a single segment
+    Generates the waveforms for a single segment.
+
+    Args:
+        start:
+            GPS time of the beginning of the testing dataset
+        stop:
+            GPS time of the end of the testing dataset
+        shifts:
+            A list of shifts in seconds. Each value corresponds to the
+            the length of time by which an interferometer's timeseries is
+            moved during one shift. For example, if `ifos = ["H1", "L1"]`
```

Comment:
I think we can jut say "the length of time by which each interferometer's timeseries will be shifted", since we're not doing multiple shifts in a single call of this function.